provider_base/services/couchdb.json

```diff
@@ -8,8 +8,8 @@
         }
     },
     "couch": {
-        "master": false,
         "port": 5984,
+        "mode": "multimaster",
```

Comment:
the default needs to remain as multimaster, or the next time you deploy it will destroy all your data and replace bigcouch with couchdb. the default for *new* nodes will be 'plain'